cmd/serverless/main.go

```diff
@@ -224,56 +224,13 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 
 	// Concurrently start heavyweight features
 	var wg sync.WaitGroup
-	wg.Add(3)
-
-	// starts trace agent
-	go func() {
-		defer wg.Done()
-		traceAgent := &trace.ServerlessTraceAgent{}
-		traceAgent.Start(config.Datadog.GetBool("apm_config.enabled"), &trace.LoadConfig{Path: datadogConfigPath}, lambdaSpanChan, coldStartSpanId)
-		serverlessDaemon.SetTraceAgent(traceAgent)
-	}()
-
-	// enable telemetry collection
-	go func() {
-		defer wg.Done()
```

Comment:
If you just add your change here, it would cut the diff down significantly, making it faster to review. 😄 

```go
	// enable telemetry collection
	go func() {
		defer wg.Done()
		if len(os.Getenv("DD_LOCAL_TEST")) > 0 {
			log.Debug("Running in local test mode. Telemetry collection HTTP route won't be enabled")
		}
		log.Debug("Enabling telemetry collection HTTP route")
		// ... etc
```

Reply:
yep good idea, I will revert the other changes! Thanks!